ablator/modules/storage/remote.py

```diff
@@ -37,6 +74,21 @@ class RemoteConfig(ConfigBase):
     exclude_chkpts: bool = False
 
     def _make_cmd_up(self, local_path: Path, destination: str):
+        """
+        Make the rsync command to upload a file to the remote storage.
```

Comment:
The `-r` used in rsync command recurses into directories, so I think this one will upload all files in the `local_path` to the destination